src/pages/settings/AppDownloadLinks.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import {ScrollView} from 'react-native';
+
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import CONST from '../../CONST';
+import {
+    Android, Apple, NewWindow, Monitor,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {
+    withLocalizePropTypes,
+} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import MenuItem from '../../components/MenuItem';
+import openURLInNewTab from '../../libs/openURLInNewTab';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+
+};
+
+const PreferencesPage = ({translate}) => {
```

Comment:
This naming is a bit confusing. This doesn't match the file name and is used elsewhere already.